bin/notifications.sh

```diff
@@ -108,9 +108,9 @@ function notification(){
 	for _LFS_ROOT_DIR in ${_LFS_ROOT_DIRS[@]}
 	do
 		_project_state_files=()
-		if $(ls "${_LFS_ROOT_DIR}/logs/"*"/"*".${_phase}.${_state}" 1> /dev/null 2>&1)
+		if $(ls "${_LFS_ROOT_DIR}/logs/"*"/"*"${_phase}.${_state}" 1> /dev/null 2>&1)
```

Comment:
Not sure if this is a "fix". No longer requiring separate state files per "run0*" means it can get all mixed up...